src/docs/default/sections/20-operationalize/10-renewal/20-deployments/linux/20-override.mdx

```diff
@@ -0,0 +1,10 @@
+<p>
+    The <inlineCode>cert-renewer@.service</inlineCode> and <inlineCode>cert-renewer@.timer</inlineCode> files define a <i>service unit template</i>.
+    Templates are never run directly,
+    but we will run an instance of them for <DocConfig>{({ name }) => name}</DocConfig> certificate renewals.
+    That service will be called <inlineCode>cert-renewer@<DocConfig>{({ linux }) => linux.systemdUnitName}</DocConfig>.service</inlineCode>,
+    which parallels the <inlineCode><DocConfig>{({ linux }) => linux.systemdUnitName}</DocConfig>.service</inlineCode> unit name
+    used by <DocConfig>{({ name }) => name}</DocConfig> on systemd-based Linux distributions.
+    Because the two units share this value in their name,
+    the <inlineCode>cert-renewer@<DocConfig>{({ linux }) => linux.systemdUnitName}</DocConfig>.service</inlineCode> renewer will attempt to reload or restart <inlineCode><DocConfig>{({ linux }) => linux.systemdUnitName}</DocConfig>.service</inlineCode> after it renews the certificate.
+</p>
```

Comment:
This works! Nice default. Will it work for some technologies with no further explanation, or should we drop a note to contribute this section if we don't have it filled in yet?

Reply:
It will work by default with some services. For example nginx's service unit has an `ExecReload=` that sends it a HUP signal. So, it needs no override!